src/components/TodoList/TodoList.tsx

```diff
@@ -1,210 +1,106 @@
 /* eslint-disable max-len */
```

Comment:
Remove the 'eslint-disable max-len' comment, as it's not necessary. If the linter complains about line length, fix the lines instead of disabling the rule.